pager.go

```diff
@@ -339,7 +352,7 @@ type PagerOptions struct {
 	// query is for any one-off URL params used by a specific endpoint.
 	// Values sent as time.Time or recurly.NullTime will be automatically
 	// converted to a valid datetime format for Recurly.
-	query query
+	Query query
```

Comment:
I'm not seeing why `query` needs to be exported -- can you help me understand?

I saw this comment below in your PR but maybe I missed where the fields were added? I did a quick pass through this, so apologies if I missed it!

> I changed PagerOptions query field to be public (Query) - add on usage record utilizies some non-standard query parameters. If this should be done different way, please let me know and I'll fix that.

Reply:
It may be my fault, but I'm not sure how non-standard filtering can be done? (Should there be another parameter or should this non-standard field be added to PagerOptions?).

Add on usage allows to filter usages list by billing_status field:
https://dev.recurly.com/docs/usage-record-object

As I library client, I was thinkin of using this filter like that:

`
query["billing_status"] = status
options := &recurly.PagerOptions{
		Query: query,
}
`
Please, let me know how this should be changed, so I can fix that.